cmake/hdf5.cmake

```diff
@@ -0,0 +1,41 @@
+#------------------------------------------------------------------------------#
+# Copyright (c) 2014 Los Alamos National Security, LLC
```

Comment:
I'm being pedantic, but this should probably be changed.

Reply:
@cmsquared: agree. I'll fix it in few minutes